src/sdp/DefaultSDP.ts

```diff
@@ -480,4 +480,76 @@ export default class DefaultSDP implements SDP {
     }
     return true;
   }
+
+  removeH264SupportFromSendSection(): DefaultSDP {
```

Comment:
I feel guilty to ask this on whats mostly code i wrote :P, but can we sprinkle some high level comments in here.  Its kinda a big blob of regex and string manipulation, which usually can be pretty fragile.

Reply:
I think more example SDP lines would be nice.

Reply:
Add comments :D